src/PhpBrew/Patches/PHP56WithOpenSSL11Patch.php

```diff
@@ -16,8 +16,11 @@ public function desc()
 
     public function match(Buildable $build, Logger $logger)
     {
-        return version_compare($build->getVersion(), '5.6') === 0
-            && version_compare($build->getVersion(), '5.6.31') >= 0 // patch only works for 5.6.31 and up
+        $buildVersion = $build->getVersion();
+        $versions = explode('.', $buildVersion);
+
+        return version_compare($versions[0].'.'.$versions[1], '5.6') === 0
```

Comment:
What if we replace it with `substr($version, 0, 4) === '5.6.'`? This way it would be clearer what is being checked and safer (otherwise, not all array indices may exist depending on the actual input).

Reply:
It's okay for me, I'll do it.

Reply:
Done!